frontend/test/pytest/test_autograph.py

```diff
@@ -1756,6 +1758,50 @@ def g():
         assert g() == 36.4
 
 
+class TestAutographInclude:
+    """Test include modules to autograph conversion"""
+
+    def test_dummy_func(self):
+        """Test dummy function branches."""
+
+        assert dummy_func(6) == 36
+        assert dummy_func(4) == 64
+
+    def test_autograph_included_module(self):
+        """Test autograph included module."""
+
+        @qjit(autograph=True)
+        def excluded_by_default(x: float, n: int):
+            for _ in range(n):
+                x = x + dummy_func(6)
+            return x
+
+        @qjit(autograph=True, autograph_include=["catalyst.utils.dummy"])
```

Comment:
nice :+1:

Reply:
Would it also work if you type `catalyst.utils`?

Reply:
yes